src/static/js/earn.js

```diff
@@ -1639,46 +3196,31 @@ async function main() {
                     ${availableStake}
                     <div class="col-sm-12 col-md-2 align-items-center text-center snob-tvl pb-10 pb-md-0">
                         <p class="m-0 font-size-12"><ion-icon name="flame-outline"></ion-icon> Pending SNOB</p>
-                        <p class="m-0 font-size-16 font-weight-regular">${(options.pending_tokens / 1e18).toFixed(6)}</p>
+                        <p class="m-0 font-size-16 font-weight-regular">${(options.pending_tokens / 1e18).toFixed(
+                          6
+                        )}</p>
                     </div>
                     ${availableUnstake}
                 </div>
             </div>
         </div>
-    </div>`;
-      $('#snob-pools-used').append(poolPrint);
+    </div>`
+      $('#snob-pools-used').append(poolPrint)
     }
   }
-  poolS3F({
-    logo_token3 : 'https://assets.coingecko.com/coins/images/13422/small/frax_logo.png?1608476506',
-    logo_token2 : 'https://raw.githubusercontent.com/ava-labs/bridge-tokens/main/avalanche-tokens/0x1C20E891Bab6b1727d14Da358FAe2984Ed9B59EB/logo.png',
-    logo_token1 : 'https://raw.githubusercontent.com/ava-labs/bridge-tokens/main/avalanche-tokens/0xde3A24028580884448a5397872046a019649b084/logo.png',
-    pool_nickname: 'pool-8',
-    pool_name: 'StableVault S3F 🌟',
-    url: null,
-    tvl: null,
-    pool_weight: null,
-    total_staked: totalStakedS3F,
-    user_pool_percent: userPool8Percent,
-    staked_pool: stakedPool8,
-    pending_tokens: pendingSNOBTokensPool8,
-    display_amount: S3FDisplayAmt,
-    approve: 'approveS3F',
-    stake: 'stakeS3F',
-    unstake: 'withdrawPool8',
-    claim: 'claimPool8',
-    icequeen_apr: pool8APR,
-    snowglobe_apr: null,
-    tvl_display: pool8tvlDisplay,
-    total_pgl: null,
-    pool_share_display: poolShareDisplay_8,
-    pool_share_display_pgl: '',
-    stake_display: ''
+  await renderPoolS3F({
```

Comment:
I replaced the **poolS3F** using this function. This function is declared later using the old function declaration syntax to hoist.